### ISSUE TITLE
INAV 2.6.1 configurator bugs

### ISSUE THREAD

**Issue by Cutbangrian** [iNavFlight/inav#6832]
Hi, I just installed inav 2.6.1 on my F722 wing.

But when it comes to accel calibration, the value keeps on 4096.

I've re flash firmware, re install the configurator, but it keep the same.
I also trying another usb cable. But it just wont work.
Never had this experience before.
![20210414_215319](https://user-images.githubusercontent.com/39833024/114736458-4f768f00-9d70-11eb-9412-ea89f46fc3bd.jpg)

**Comment by Cutbangrian** [comment]
ATTENTION...

The problem is, when step 1 is completed then it will turned into blue.
But when I press the button to the step 2, 

The step one go back to grey colour.

Never had this experience before. Please help

This is the pict
![Uploading 20210414_215325.jpg…]()

**Comment by Jetrell** [comment]
Did you try setting the controller to `Defaults` in the CLI ?  
Then remove the `ACCZERO` and `ACCGAIN` settings from your Diff files (# master) before you reload them into the CLI.

After this you should be able to complete the calibration.

**Comment by Cutbangrian** [comment]
> Did you try setting the controller to `Defaults` in the CLI ?
> Then remove the `ACCZERO` and `ACCGAIN` settings from your Diff files (# master) before you reload them into the CLI.
> 
> After this you should be able to complete the calibration.

This stil not clear to me, I'm sorry.
I didnt touch anything in CLi

I just bought this fc and installed the latest firmware 2.6.1 with inav configurator 2.6.

The first thing I did is accel calibration.
But just like I described before, the value wont changes.

If value on gui changes, I can entry the value manually into cli.

But it's not. The value remain 4096 even in gui and cli.
Means not calibrated.


So I need to delete acc zero and acc gain in diff file?
